admin-ui/app/locales/en/translation.json

```
@@ -977,9 +1079,25 @@
+    "add_claims": "Add claims",
+    "add_email": "Add email",
+    "search_clients": "Search clients...",
+    "search_scopes": "Search scopes...",
+    "localized_json_format": "JSON format: {\"en\": \"value\", \"es\": \"valor\"}",
+    "enter_client_name": "Enter client name",
+    "enter_description": "Enter description",
+    "select_grant_types": "Select grant types"
   },
   "titles": {
+    "actions": "Actions",
+    "id_token": "ID Token",
```

Biome (2.1.2)

[error] The key id_token was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)